apps/server/src/modules/tool/tool.module.ts

```diff
@@ -1,14 +1,32 @@
 import { Module } from '@nestjs/common';
 import { LtiToolRepo, PseudonymsRepo } from '@shared/repo';
-import { Lti11ResponseMapper } from '@src/modules/tool/mapper/lti11-response.mapper';
 import { LoggerModule } from '@src/core/logger';
+import { AuthorizationModule } from '@src/modules/authorization';
+import { LtiToolSortingMapper } from '@shared/repo/ltitool/lti-tool-sorting.mapper';
+import { Lti11ResponseMapper } from './mapper/lti11-response.mapper';
+import { LtiToolMapper } from './mapper/lti-tool.mapper';
+import { LtiToolController } from './controller/lti-tool.controller';
+import { ToolController } from './controller/tool.controller';
+import { LtiToolUc } from './uc/lti-tool.uc';
 import { Lti11Uc } from './uc/lti11.uc';
 import { LtiRoleMapper } from './mapper/lti-role.mapper';
 import { Lti11Service } from './service/lti11.service';
 import { UserModule } from '../user';
 
 @Module({
-	imports: [UserModule, LoggerModule],
-	providers: [Lti11Service, Lti11Uc, LtiRoleMapper, Lti11ResponseMapper, LtiToolRepo, PseudonymsRepo],
+	controllers: [LtiToolController, ToolController],
+	imports: [UserModule, LoggerModule, AuthorizationModule],
+	providers: [
+		Lti11Service,
+		Lti11Uc,
+		LtiRoleMapper,
+		Lti11ResponseMapper,
+		LtiToolUc,
+		LtiToolRepo,
+		PseudonymsRepo,
+		LtiToolMapper,
+		LtiToolSortingMapper,
+	],
+	exports: [LtiToolRepo],
```

Comment:
this is already a big step forward, but it would be a lot better if the repo was not directly exposed, but some kind of service. It would also be an improvement to split this into two modules, one containing the controller and uc, and the other the rest.